Mathlib/Algebra/Order/Monoid/Unbundled/Basic.lean

```diff
@@ -283,6 +291,24 @@ theorem mul_right_cancel'' [MulRightReflectLE α] {a b c : α}
   haveI := mulRightMono_of_mulRightStrictMono α
   rw [le_antisymm_iff, eq_true (mul_le_mul' hac hbd), true_and, mul_le_mul_iff_of_ge hac hbd]
 
+@[to_additive]
+lemma mul_left_inj_of_comparable [MulRightStrictMono α] {a b c : α} {h : b ≤ c ∨ c ≤ b} :
```

Comment:
```suggestion
lemma mul_left_inj_of_comparable [MulRightStrictMono α] {a b c : α} (h : b ≤ c ∨ c ≤ b) :
```

Reply:
Good point: I am not sure why I made these assumptions implicit in the first place.